k8s/schema_cache.go

```diff
@@ -177,7 +177,7 @@ func (sc *SchemaCache) fetchAndSetSchema() error {
 }
 
 func (sc *SchemaCache) fetchAndSetResources() error {
-	serverResources, err := sc.clientset.ServerResources()
+	serverResources, err := sc.clientset.ServerPreferredResources()
```

Comment:
@moshloop What do we want to do here ? is ServerPreferredResources a correct replacement for ServerResources

Reply:
should be fine